packages/@webex/internal-plugin-metrics/src/rtcMetrics/index.ts

```diff
@@ -15,6 +15,22 @@ const parseJsonPayload = (payload: any[]): any | null => {
   }
 };
 
+interface IdType {
+  meetingId?: string;
+  callId?: string;
+}
+
+interface IMetricsAttributes {
```

Comment:
Do we have any types or interface file where this can be moved?

Reply:
Moved to `packages/@webex/internal-plugin-metrics/src/metrics.types.ts`